regulations/unified_metrics/environment/biodiversity_sensitive_sites.md

```diff
@@ -79,22 +95,30 @@ biodiversity
 
 ### examples_en
 
-
+If you have company operations in or near biodiversity-sensitive sites, please report the number of operative sites as well as
```

Comment:
This is not an example of an answer, this is the question. Please provide actual examples.